app/scripts/i18n/locale_en.json

```diff
@@ -18,10 +18,10 @@
     "map.filter": "Map filters",
     "map.legend": "Legend",
     "passrate-perc": "Passrate (%):",
-    "chart.dial-title": "Change since 2012",
+    "chart.dial-title": "CHANGE SINCE",
```

Comment:
We can do the all-caps with css `text-transform: uppercase`, I think we should keep translations as clean and normalized as we can.